ApplicationLibCode/ProjectDataModel/RimEclipseStatisticsCaseEvaluator.cpp

```diff
@@ -310,14 +311,17 @@ void RimEclipseStatisticsCaseEvaluator::evaluateForResults( const QList<ResSpec>
         // Microsoft note: On Windows, the maximum number of files open at the same time is 512
         // http://msdn.microsoft.com/en-us/library/kdfaxaay%28vs.71%29.aspx
 
+        std::vector<RiaDefines::ResultCatType> categoriesToExclude;
+        if ( !m_clearGridCalculationMemory ) categoriesToExclude.push_back( RiaDefines::ResultCatType::GENERATED );
+
         for ( size_t caseIdx = 0; caseIdx < m_sourceCases.size(); caseIdx++ )
         {
             RimEclipseCase* eclipseCase = m_sourceCases.at( caseIdx );
 
```

Comment:
`for (RimEclipseCase* eclipseCase : sourceCases)` maybe?

Reply:
Fixed